.env.example

```diff
@@ -1,3 +0,0 @@
-DEFIPULSE_KEY='SDK_KEY_HERE'
```

Comment:
Why were these removed from .env.example?

Reply:
my bad sorry. Will undo

Reply:
All good, I assume it's the result of removing .env files from git to make sure private details weren't committed which 100% makes sense.